src/pages/login.js

```diff
@@ -1,5 +1,88 @@
+import Background from "../assets/universe_background.jpeg";
+import Logo from "../assets/spacecenter1.svg";
+import { useNavigate } from "react-router-dom";
+import { post } from "../Utils";
+
 function Login() {
-  return <h1>Login</h1>;
+  const navigate = useNavigate();
+
+  const current_session = {
+    user: "",
+    token: "",
+  };
+
+  const handleSubmit = async (event) => {
+    const mail = document.getElementById("email").value;
+    const pass = document.getElementById("password").value;
+
+    if (mail && pass) {
+      await post(`auth/login`, {
+        email: mail,
+        password: pass,
+      }).then((response) => {
+        current_session["user"] = response["user"];
+        current_session["token"] = response["response"];
```

Comment:
Why are you setting these values on `current_session`, if you're only using them to set local storage?  You could  move the `localStorage.setItem` statements into the `then` callback, and remove `current_session`.